### ISSUE TITLE
Video duration is hidden for solid and blurred options

### ISSUE THREAD

**Issue by ghost** [domdomegg/hideytthumbnails-extension#32]
I guess this is not intended behavior of this extension

Video duration is not related to clickbaity behavior

![image](https://github.com/domdomegg/hideytthumbnails-extension/assets/86713661/b0efb116-bb17-442e-8448-535a5df9d2d6)

**Comment by ghost** [comment]
I might be wrong about

I'm not sure whether it is a recent update, but it might be a purely new YouTube behavior

**Comment by bandhan-majumder** [comment]
@ghost well it's a valid issue I guess. Cause the duration is not related to clickability right.

**Comment by vakokako** [comment]
yep, this would be great, the solid color option is the best, apart from this disadvantage

**Comment by domdomegg** [comment]
Via email:

> Howdy!
> I love the extension. Would it be possible for you to ship an option that kept the length of the videos visible while hiding the thumbnail?
> (I'm using it with blurred, but TBH, I really like hiding the thumbnail fully.)

**Comment by domdomegg** [comment]
On the above, I'd be very happy to receive a PR for this change but unfortunately I don't have time to implement it myself right now.

You'd want to edit https://github.com/domdomegg/hideytthumbnails-extension/blob/master/inject.js to tweak with the CSS rules for the `hidden` and `blurred` options to find a configuration that works for hiding the thumbnail while not hiding the video duration.
